src/Strategy/Maven/PluginStrategy.hs

```diff
@@ -50,7 +50,7 @@ buildGraph PluginOutput{..} = run $
         , dependencyName = artifactGroupId <> ":" <> artifactArtifactId
         , dependencyVersion = Just (CEq artifactVersion)
         , dependencyLocations = []
-        , dependencyEnvironments = if "test" `elem` artifactScopes then [EnvTesting] else []
+        , dependencyEnvironments = [EnvTesting | "test" `elem` artifactScopes]
```

Comment:
I think this is a scenario where the previous `if else` block is more clear. Can we ignore warnings for this issue too?

Reply:
This is already covered by `Use list comprehesion`.  We can just revert this one.

Reply:
Ah, didn't realize this was covered by that ignore.